### PR TITLE
Feature carousel

### PR DESCRIPTION
**Pull request by jengle-dev**

### CODE REVIEW

**Review by jengle-dev — COMMENTED** [review]
please reject anything I'm deleting and check that my additions make sense. when I pulled down I'm not sure what merged and didn't. sorry. @rebeccalehew @juliaguas17

**Review by juliaguas17 — APPROVED** [review]

**Comment by jengle-dev** [comment]
never mind. i'm fixing it

File: client/src/index.css [hunk 1]
```diff
@@ -33,28 +33,34 @@ body {
   font-family: 'Lora', serif;
   font-family: 'Old Standard TT', serif;
   font-family: 'Roboto Slab', serif;
+  background-color: var(--mosseyRock);
```
Comment (rebeccalehew) [inline client/src/index.css:36]: should it be --mossyRock ???

File: client/src/components/Footer.js [hunk 2]
```diff
@@ -1,19 +1,33 @@
...
+<<<<<<< HEAD
+    <div background="">Footer</div>
+    <img src={gitHub} alt='GitHub' />
+    <img src={insta} alt='Instagram' />
+=======
       <img src={brownMossy}/>
       <img src={brownToad}/>
       <img src={greenMossy}/>
       <img src={greenToad}/>
+>>>>>>> 9cb780e1237aa58c8eb0969dddfba8775c2e19be
```
Comment (rebeccalehew) [inline client/src/components/Footer.js:28]: @jengle-dev What is this line for?

**Pull request closed without merging**